paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -65,6 +65,27 @@ internal sealed class PaymentSelection : Parcelable {
         }
     }
 
+    @Parcelize
+    data class ExternalPaymentMethod(
+        val type: String,
+        val label: String,
+        @DrawableRes val iconResource: Int,
+        val lightThemeIconUrl: String?,
```

Comment:
This type is not super realistic which makes me kind of sad. In actuality, the lightThemeIconUrl is never null and the iconResource is never there.

The PR to create a PaymentMethodIcon type would fix this, but still not sure if it's worth it as obviously this is much simpler...

Reply:
I think we probably have to keep the lightThemeIconUrl as nullable, but we could probably remove the iconResource, and make a comment that in practice the lightThemeIconUrl shouldn't be null.

Reply:
Added some comments, thanks! I agree that we could remove the iconResource, but then I think we have to set it to 0 in multiple places. I prefer threading it through because if we update the value at some point, we just have to update it in one place